source/references.adoc

```diff
@@ -1,19 +1,26 @@
 [appendix]
 = References
 
-Below are references to various topics of interest around RPMs, RPM Packaging,
-and RPM Building. Some of these will be advanced and extend far beyond the
-introductory material included in this guide.
+Below are references to various topics of interest around RPMs, RPM Packaging, and RPM Building. Some of these will be advanced and extend far beyond the introductory material included in this guide.
```

Comment:
Below are references to various topics of interest around RPMs, RPM packaging, and RPM building. Some of these are advanced and extend far beyond the introductory material included in this guide.